wandb/sdk/wandb_settings.py

```diff
@@ -507,7 +507,8 @@ def _validate_mode(self, value):
             "run",
             "offline",
             "online",
-            "noop",
+            "noop",  # deprecated
```

Comment:
@raubitsj we don't have anyone actually using "noop" do we?  I think we can just remove it.